app/src/main/java/pl/szczodrzynski/edziennik/data/api/edziennik/librus/data/api/LibrusApiTeacherFreeDays.kt

```diff
@@ -36,12 +36,12 @@ class LibrusApiTeacherFreeDays(override val data: DataLibrus,
                 val id = teacherAbsence.getLong("Id") ?: return@forEach
                 val teacherId = teacherAbsence.getJsonObject("Teacher")?.getLong("Id")
                         ?: return@forEach
-                val type = teacherAbsence.getJsonObject("Type").getLong("Id") ?: return@forEach
+                val type = "-1".toLong()
                 val name = data.teacherAbsenceTypes.singleOrNull { it.id == type }?.name
                 val dateFrom = Date.fromY_m_d(teacherAbsence.getString("DateFrom"))
                 val dateTo = Date.fromY_m_d(teacherAbsence.getString("DateTo"))
-                val timeFrom = teacherAbsence.getString("TimeFrom")?.let { Time.fromH_m_s(it) }
-                val timeTo = teacherAbsence.getString("TimeTo")?.let { Time.fromH_m_s(it) }
+                val timeFrom = null
+                val timeTo = null
```

Comment:
Inline these variables. It's not needed to create a variable for a constant value, so just put them in the constructor of the object below. Also, `"-1".toLong()` is just `-1L`.

Reply:
Im not sure what do you mean, something like this?
```
teacherAbsences?.forEach { teacherAbsence ->
                val id = teacherAbsence.getLong("Id") ?: return@forEach
                val teacherId = teacherAbsence.getJsonObject("Teacher")?.getLong("Id")
                        ?: return@forEach
                val name = data.teacherAbsenceTypes.singleOrNull { it.id == -1L }?.name
                val dateFrom = Date.fromY_m_d(teacherAbsence.getString("DateFrom"))
                val dateTo = Date.fromY_m_d(teacherAbsence.getString("DateTo"))

                val teacherAbsenceObject = TeacherAbsence(
                        profileId = profileId,
                        id = id,
                        type = -1L,
                        name = name,
                        dateFrom = dateFrom,
                        dateTo = dateTo,
                        timeFrom = null,
                        timeTo = null,
                        teacherId = teacherId
                )
```

Reply:
yes